<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>563</width>
    <height>365</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Serial Bluetooth AT Commands</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/icon/Bluetooth</normaloff>:/images/icon/Bluetooth</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lblSelectPortCom">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select Port Com</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbPortsCom">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>521</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btConnect">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>70</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbBaudRate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblSelectBaudRate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select Baud Rate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btRefresh">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>30</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/images/icon/Refresh</normaloff>:/images/icon/Refresh</iconset>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabTerminal">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>571</width>
      <height>301</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tabHC05">
     <attribute name="title">
      <string>HC-05</string>
     </attribute>
     <widget class="QGroupBox" name="gbMainContent_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>541</width>
        <height>251</height>
       </rect>
      </property>
      <property name="title">
       <string>AT Bluetooth Commands</string>
      </property>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>30</y>
         <width>141</width>
         <height>91</height>
        </rect>
       </property>
       <property name="title">
        <string>Current Informations</string>
       </property>
       <widget class="QTextBrowser" name="taReception_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>121</width>
          <height>61</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>181</width>
         <height>91</height>
        </rect>
       </property>
       <property name="title">
        <string>Change Name</string>
       </property>
       <widget class="QLineEdit" name="txtName_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>161</width>
          <height>20</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;No space, no special char.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Only : Alphabetic and numeric without decimal.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btSetName_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btClearName_2">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>60</y>
          <width>71</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>181</width>
         <height>91</height>
        </rect>
       </property>
       <property name="title">
        <string>Change Pin</string>
       </property>
       <widget class="QLineEdit" name="txtPin_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>161</width>
          <height>20</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;No space, no special char.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Only : Alphabetic and numeric without decimal.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
       </widget>
       <widget class="QPushButton" name="btSetPin_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btClearPin_2">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>60</y>
          <width>71</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_9">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>30</y>
         <width>181</width>
         <height>91</height>
        </rect>
       </property>
       <property name="title">
        <string>Change Baudrate</string>
       </property>
       <widget class="QPushButton" name="btSetBaudRate_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cbBaudRateSerial_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>161</width>
          <height>22</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;If you change BaudRate,&lt;br/&gt;you must edit it in Arduino.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btBaudRateDefault_2">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>60</y>
          <width>71</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Default</string>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="btVersion_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>200</y>
         <width>181</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Get Firmware Revision</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btVersion_3">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>140</y>
         <width>141</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Get Name</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btVersion_4">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>170</y>
         <width>141</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Get Address</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btVersion_5">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>200</y>
         <width>141</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Get BaudRate</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btVersion_6">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>140</y>
         <width>181</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Reset and Exit AT</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btVersion_7">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>170</y>
         <width>181</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Restore Factory</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tabHC06">
     <attribute name="title">
      <string>HC-06</string>
     </attribute>
     <widget class="QGroupBox" name="gbMainContent">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>541</width>
        <height>251</height>
       </rect>
      </property>
      <property name="title">
       <string>AT Bluetooth Commands</string>
      </property>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>20</y>
         <width>141</width>
         <height>211</height>
        </rect>
       </property>
       <property name="title">
        <string>Current Informations</string>
       </property>
       <widget class="QTextBrowser" name="taReception">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>121</width>
          <height>181</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>181</width>
         <height>91</height>
        </rect>
       </property>
       <property name="title">
        <string>Change Name</string>
       </property>
       <widget class="QLineEdit" name="txtName">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>161</width>
          <height>20</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;No space, no special char.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Only : Alphabetic and numeric without decimal.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btSetName">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btClearName">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>60</y>
          <width>71</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>181</width>
         <height>91</height>
        </rect>
       </property>
       <property name="title">
        <string>Change Pin</string>
       </property>
       <widget class="QLineEdit" name="txtPin">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>161</width>
          <height>20</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;No space, no special char, no char.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Only : Numeric without decimal.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Range : 0000 - 9999&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Example : 1247 OR 2148 OR 9751 ETC ...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
       </widget>
       <widget class="QPushButton" name="btSetPin">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btClearPin">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>60</y>
          <width>71</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="gbBaudRate">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>30</y>
         <width>181</width>
         <height>91</height>
        </rect>
       </property>
       <property name="title">
        <string>Change Baudrate</string>
       </property>
       <widget class="QPushButton" name="btSetBaudRate">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cbBaudRateSerial">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>161</width>
          <height>22</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;If you change BaudRate,&lt;br/&gt;you must edit it in Arduino.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btBaudRateDefault">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>60</y>
          <width>71</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Default</string>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="btVersion">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>210</y>
         <width>181</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Get Firmware Revision</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Terminal</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>541</width>
        <height>231</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(186, 218, 85);</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;### Terminal Ingoing ###&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtCustomCommand">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>461</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QPushButton" name="btSendCustomCommand">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>250</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btDisconnect">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>595</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuArduino_Code">
    <property name="title">
     <string>Documentations</string>
    </property>
    <addaction name="actionArduino"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuArduino_Code"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icon/Delete</normaloff>:/images/icon/Delete</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionArduino">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icon/Arduino</normaloff>:/images/icon/Arduino</iconset>
   </property>
   <property name="text">
    <string>Arduino</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
